pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -38,7 +38,7 @@ const (
 	// Additional caches can be created, but only as copies of default cache.
 	ServiceTypeCache ServiceType = "Cache"
 
-	// Deploys Infinispan to act like a data grid.
+	// Deploys Infinispan to act like a Data Grid.
```

Comment:
do we need it uppercase?
```suggestion
	// Deploys Infinispan to act like a data grid.
```

Reply:
Yes, sure, it more type than name in this case